src/Commands/RunScan.spec.ts

```diff
@@ -67,4 +88,123 @@ describe('RunScan', () => {
       expect(act).toThrowError(SyntaxError);
     });
   });
+
+  describe('handler', () => {
+    let runScan: RunScan;
+    let mockedRestScans: RestScans;
```

Comment:
Tests should not depend on implementations
`ts-mockito` allows to mock interfaces, there's also no need to re-assign mock
```suggestion
    const mockedRestScans: Scans = mock<Scans>();
```

Reply:
sure, agree